### DISCUSSION TITLE
[Build] 2.2.0 fail to build with FreeBSD/Mariadb because of #631 not in release

### DISCUSSION THREAD

**Discussion by karolyi** [PyMySQL/mysqlclient#643] [Build issue]
### What OS and which version do you use?

FreeBSD

### How did you installed mysql client library?

```bash
pkg install databases/mariadb1011-client
```

### Output from `pkg-config --cflags --libs mysqlclient`

The usual suspect, pkg-config will miss the mariadb dependency because it's to be found as `libmariadb` not as `mariaddb`. #631 fixes this, but it's not released, hence 2.2.0 will fail to build in my scenario.

**Comment by methane** [comment]
You can build it by manually setting env var.
See https://github.com/PyMySQL/mysqlclient#customize-build-posix

**Reply by karolyi** [reply]
I solved it by temporarily renaming the libmariadb.pc to mariadb.pc so pkg-config would find it. Afterwards pip will cache the built wheel so no renaming necessary again, unless the next version still has this error.
